openstack_controller/datadog_checks/openstack_controller/openstack_controller.py

```diff
@@ -234,6 +248,12 @@ def get_stats_for_single_hypervisor(self, hyp, custom_tags=None,
             'virt_type:{}'.format(hyp['hypervisor_type']),
             'status:{}'.format(hyp['status']),
         ]
+
+        # add hypervisor project names as tags
+        project_names = hyp_project_names.get(hyp_hostname, {})
```

Comment:
The default should be a list, no? Or a set, but it doesn't really matter in this case.

Reply:
A set otherwise we end up with duplicated project names